readme.md

```diff
@@ -40,6 +40,7 @@ For an overview about the **features** and **tradeoffs** of styled-jsx you may w
   * [Can I return an array of components when using React 16?](#can-i-return-an-array-of-components-when-using-react-16)
   * [Styling third parties / child components from the parent](#styling-third-parties--child-components-from-the-parent)
   * [Some styles are missing in production](https://github.com/zeit/styled-jsx/issues/319#issuecomment-349239326)
+  * [Usage with Rollup](#usage-with-rollup)
```

Comment:
This title might be confusing because it is about bundling a single library - if people use Rollup to build the entire app probably they'd need to include styled-jsx and react in the bundle. Maybe we can rename it to "Build a component library with styled-jsx"

Reply:
I wasn't sure about that either. Sure, I will name it more precisely.